appveyor.yml

```diff
@@ -1,14 +1,18 @@
 environment:
-  DATADEPS_ALWAY_ACCEPT: True
```

Comment:
don't delete this.
It is best to have this in the CI script; so CI works.
Rather than putting it into the tests itself, as that will make it autoaccept downloads for users, if they run the tests.
And once downloaded they will not be prompted again outside of tests